docs/pages/build/building-from-github.mdx

```diff
@@ -214,10 +214,41 @@ On your project's GitHub settings page in the Expo dashboard, you can click the
 a build trigger row to disable, edit, or delete the trigger.
 
 <ImageSpotlight
-  alt="Manage GitHub build trigger with the options button"
-  src="/static/images/eas-build/build-from-github/managing-build-triggers.png"
+  alt="The options button on the build trigger row"
+  src="/static/images/eas-build/build-from-github/edit-trigger-option.png"
```

Comment:
```suggestion
  src="/static/images/eas-build/build-from-github/edit-trigger-option.png"
  style={{ maxWidth: 320 }}
```

Let's add a width here. Output after this:
![CleanShot 2024-04-30 at 19 08 24@2x](https://github.com/expo/expo/assets/10234615/03a026cd-e438-46c5-ad75-51e74f89fcae)